cmd/snap/cmd_get.go

```diff
@@ -77,6 +78,19 @@ func init() {
 		})
 }
 
+func getDottedKeys(path []string, input map[string]interface{}) []string {
+	var paths []string
+	for k, v := range input {
+		p := append(path, k)
```

Comment:
Fair point, I've removed this function altogether though for the new logic we discussed.